packages/components/tests/integration/components/hds/form/textarea/field-test.js

```diff
@@ -15,11 +15,7 @@ module('Integration | Component | hds/form/textarea/field', function (hooks) {
     resetOnerror();
   });
 
-  test('it renders the component', async function (assert) {
-    await render(hbs`<Hds::Form::Textarea::Field />`);
-    assert.dom('textarea').exists();
-  });
-  test('it should render the input with a specific CSS class', async function (assert) {
+  test('it renders the component with a specific CSS class', async function (assert) {
```

Comment:
```suggestion
  test('it renders the component with a CSS class that matches the component name', async function (assert) {
```

Reply:
In the PR description, there's a bullet point:

> there are a few exceptions here and there but in context it made more sense to leave them be. 

This was, from what I saw, mostly concerning form elements.